net-misc/gns3-server/files/gns3-server-2.1.16-typing.patch

```diff
@@ -0,0 +1,10 @@
+--- a/requirements.txt	2019-03-20 20:41:44.000000000 -0700
++++ b/requirements.txt	2019-04-23 12:16:29.564870881 -0700
+@@ -6,6 +6,6 @@
+ raven>=5.23.0
+ psutil>=3.0.0
+ zipstream>=1.1.4
+-typing>=3.5.3.0 # Otherwise yarl fails with python 3.4
++typing>=3.5.3.0;python_version<"3.5" # Otherwise yarl fails with python 3.4
```

Comment:
We don't support py3.4 anymore.

Reply:
i am directly merging a patch from upstream, so i am not editing its text in any way.. it still fixed the runtime issue here.

But looks like typing is dropped in that latest release 2.1.20 so i'll just drop this and bump all the ebuilds